cyder/settings/local.py-dist

```diff
@@ -29,32 +29,49 @@ MIDDLEWARE_CLASSES = (
 
 MANAGERS = ADMINS
 
-DATABASES = {
+DATABASES = {  # databases used through Django
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': '',
-        'USER': '',
-        'PASSWORD': '',
         'HOST': 'localhost',
         'PORT': '',
+        'NAME': 'FIXME',
+        'USER': 'FIXME',
+        'PASSWORD': '',
         'OPTIONS': {
             'init_command': 'SET storage_engine=InnoDB',
-            'charset' : 'utf8',
+            'charset': 'utf8',
             'use_unicode' : True,
```

Comment:
you've fixed this whitespace before the `:` for `charset`, but it's still there for `use_unicode` here, and below